android/src/com/frostwire/android/gui/activities/BuyActivity.java

```diff
@@ -79,6 +79,7 @@ private void purchaseProduct(int tagId) {
     @Override
     protected void initToolbar(Toolbar toolbar) {
         toolbar.setTitle(getActionBarTitle());
+        toolbar.setTitleMarginStart(20);
```

Comment:
Actually it might be, let me double check. I was looking at BuyActivity first and left it there when it worked. Thank you for noticing.